tests/phpunit/tests/blocks/wpGetBlockPatterns.php

```diff
@@ -1,35 +1,55 @@
 <?php
 /**
- * Tests for _wp_get_block_patterns.
+ * Tests for WP_Theme::get_block_patterns.
  *
  * @package WordPress
  * @subpackage Blocks
  * @since 6.4.0
  *
  * @group blocks
  *
- * @covers ::_wp_get_block_patterns
+ * @covers WP_Theme::get_block_patterns
  */
-class Tests_Blocks_WpGetBlockPatterns extends WP_UnitTestCase {
+class Tests_Blocks_WPThemeGetBlockPatterns extends WP_UnitTestCase {
+
+	/**
+	 * Test helper to access the private get_pattern_cache method of a theme.
+	 *
+	 * @param WP_Theme $wp_theme A WP_Theme object.
+	 * @return array|false Returns an array of patterns if cache is found, otherwise false.
+	 */
+	public function get_pattern_cache( $wp_theme ) {
```

Comment:
Nit-pick: This doesn't need to be public, so better make it private.

Reply:
Handled in [2ff2d69](https://github.com/WordPress/wordpress-develop/pull/5533/commits/2ff2d6990c4564cf12a58a9988ec623c17cd1312)